vars/githubPr.groovy

```diff
@@ -97,18 +99,66 @@ def getHistoryText(builds) {
   return "### History\n${list}"
 }
 
+def getTestFailuresMessage() {
+  def failures = testUtils.getFailures()
+  if (!failures) {
+    return ""
+  }
+
+  def messages = []
+
+  failures.take(5).each { failure ->
+    messages << """
+---
+
+### [Test Failures](${env.BUILD_URL}testReport)
+<details><summary>${failure.fullDisplayName}</summary>
+
+[Link to Jenkins](${failure.url})
+
+```
+${failure.stdOut}
+```
+</details>
+
+---
+    """
```

Comment:
Unfortunately can't use `stripIndent()` here because the `stdOut` is not indented